packages/loader/container-loader/src/test/deltaManager.spec.ts

```diff
@@ -81,7 +81,7 @@ describe("Loader", () => {
 					noopCountFrequency,
 				);
 
-				await deltaManager.attachOpHandler(0, 0, 1, {
```

Comment:
maybe add a var, like `const DefaultTermValue = 1 as const`, and a comment on that value. you could do this once per package and not export it.